### DISCUSSION TITLE
the button doesn't work

### DISCUSSION THREAD

**Discussion by wwnrfhu** [maximmax42/Discord-CustomRP#235] [Q&A] [answered]
the button where I paste the link doesn't work.
![CustomRP_0atGvj9Mnu](https://github.com/maximmax42/Discord-CustomRP/assets/119198529/97e0e498-3cfd-4bfa-ba5a-ad81714dd6f8)
I press the button and nothing happens. I asked my friends to press it, nothing happens.
![Discord_izAKvJtSeQ](https://github.com/maximmax42/Discord-CustomRP/assets/119198529/d5933878-4251-4e9d-9c2c-9236822df0d2)

**Comment by maximmax42** [comment, marked as answer]
The problem is on Discord's side, can't do anything about it.
